### PR TITLE
Extend theme selectors for whitelabel theme requests

### LINKED ISSUE #2164: [FR]: See green button when clicking green order line selection
**Is your feature request related to a problem? Please describe.**
It seems counter-intuitive that clicking on the green bits in the orderbook make the red button appear.

https://user-images.githubusercontent.com/35845239/206088607-7c8ab6f2-b2b5-49d6-96c0-5f9787f82c22.mp4

**Describe your solution**
Make sell button green, to match the color of the orderbook selection
Make buy button red to match the color of the orderbook selection

### PR DESCRIPTION
**Pull request by smk762**
Closes #2164 

image

image

### CODE REVIEW

**Comment by smk762** [comment]
@Canialon I'll use this branch to extend the themes as required to cater for requested changes by whitelabel projects. Please let me know if any more cases are found that need a new selector.

**Comment by Canialon** [comment]
> @Canialon I'll use this branch to extend the themes as required to cater for requested changes by whitelabel projects. Please let me know if any more cases are found that need a new selector.

Thanks a lot! At least what I remember for now are 

- [ ] all close/cancel buttons 
- [ ] input fields for wallet creation/importing/login
- [ ] text hover effect for price column at portfolio->assets list
- [ ] change send/receive/swap icons on wallet page to this:
image

- [ ] all pop-up windows ( settings, faq, send, swap details, transportation details, faucet etc)

I will add some more tonight

**Comment by smk762** [comment]
https://user-images.githubusercontent.com/35845239/206187809-f78705db-2839-4995-85ce-4ec05839fc5a.mp4

**Comment by Canialon** [comment]
On this branch filters started opening like this on dex->pro->orders

https://user-images.githubusercontent.com/77272165/206928699-b883553e-8dd5-417f-8786-2cfbdfe625d7.mov

File: atomic_defi_design/Dex/Components/DexCancelButton.qml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import QtQuick 2.15
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexCancelButton.qml:1]: Name it just CancelButton

**Review by ghost — DISMISSED** [review]

**Comment by smk762** [comment]
> On this branch filters started opening like this on dex->pro->orders

Fixed

**Comment by smk762** [comment]
> Separate colour for seed words on seed confirmation image
> 
> color for creating new wallet -> warning image
> 
> for wallet-> coin hover effect image
> 
> borderline for simple -> trade/orders/history image
> 
> Filter design fixes:
> 
> image
> 
> image

Moved to separate issue

**Comment by smk762** [comment]
@Canialon this is ready for re-review. Some items I moved to a separate issue for later. Please give the app a once over on both lite and dark themes to make sure nothing looks weird.

**Comment by Canialon** [comment]
The theme extension looks good now, need just update the `Binance - Dark` json file. Will do it today

**Comment by smk762** [comment]
ready for review

**Review by Canialon — APPROVED** [review]

**Pull request merged**